Source/Private/LevelStatsCollector.cpp

```diff
@@ -1,24 +1,65 @@
 ﻿#include "LevelStatsCollector.h"
 
+#include "LevelStatsCollectorState.h"
+
 #include <Components/SceneCaptureComponent2D.h>
+#include <Engine/Engine.h>
 #include <Engine/LevelBounds.h>
 #include <Engine/TextureRenderTarget2D.h>
 #include <ImageUtils.h>
 
+FCustomPerformanceChart::FCustomPerformanceChart( const FDateTime & in_start_time, const FString & in_chart_label, const FString & in_output_path ) :
+    FPerformanceTrackingChart( in_start_time, in_chart_label ),
+    CustomOutputPath( in_output_path )
+{
+}
+
+void FCustomPerformanceChart::DumpFPSChartToCustomLocation( const FString & in_map_name )
```

Comment:
pass the map_name as a `FStringView` (by copy)

Reply:
no `in_` or `out_` prefix

Reply:
But done, still.